Project.toml

```diff
@@ -1,14 +1,15 @@
 name = "LibCURL"
 uuid = "b27032c2-a3e7-50c8-80cd-2d36dbcbfd21"
-version = "0.5.2"
+version = "0.6.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibCURL_jll = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
+MozillaCACerts_jll = "14a3606d-f60d-562e-9121-12d972cd8159"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "0.7, 1.0"
+LibCURL_jll = "7.70"
+MozillaCACerts_jll = "2020"
```

Comment:
If you want to always allow the latest certificate, I _think_ automerge is happy if you don't specify a compat bound for a jll package, but I need to double check this claim

Reply:
I just added this as it seemed best to have a minimum defined. I will admit this minimum is rather arbitrary

Reply:
The fact is that this isn't just a minimum but also a maximum: only this year certificates will work with this release

Reply:
Ah yes, you're right as this is working as a major version number. I'll update to: `>=2020`